backend/spec/controllers/spree/admin/reports_controller_spec.rb

```diff
@@ -9,24 +9,16 @@
     it 'should contain sales_total' do
       expect(Spree::Admin::ReportsController.available_reports.keys.include?(:sales_total)).to be true
     end
-
-    it 'should have the proper sales total report description' do
-      expect(Spree::Admin::ReportsController.available_reports[:sales_total][:description]).to eql('Sales Total For All Orders')
-    end
```

Comment:
It may be worth replacing this test with a view spec, but I'm uncertain how much value testing translations provides in this context.